web/gds-user-ui/src/components/RegistrationForm/InvalidFormPrompt.tsx

```diff
@@ -45,7 +45,10 @@ function InvalidFormPrompt({
                 <Trans>
                   If you continue, your changes will be lost. To save your changes, click Cancel and
                   then click on the{' '}
-                {isNextStep ? <Text as="span" fontWeight={'bold'} whiteSpace={'break-spaces'}>Save & Next</Text> : <Text as="span" fontWeight={'bold'} whiteSpace={'break-spaces'}>Save & Previous</Text>} button.
+                  <Text as="span" fontWeight={'bold'} whiteSpace={'break-spaces'}>
+                    {nextStepBtnContent}
```

Comment:
During QA, Edwin stated the text could state both button options and to handle this change in the future.

Reply:
i make this update during that call and forgot to remove it. I think you have a PR that should display this part as you suggested.

Reply:
I've merged that PR, so this should be removed.